proto/types/v1/localhost_config.proto

```diff
@@ -140,6 +140,7 @@ message ParseConfig {
   Time timestamp = 1;
   Message message = 2;
   Level level = 3;
+  bool detect_timestamp = 4;
```

Comment:
Let's make this a:
```
message KV {
   bool detect_timestamps = 1;
   bool detect_durations = 2;
}
```

instead, it'll scale better over time

Reply:
got it